composer.json

```
@@ -11,14 +11,14 @@
     ],
     "require": {
         "php": "^8.0",
-        "symfony/lock": "^6.0 || ^7.0",
-        "symfony/cache": "^6.0 || ^7.0",
-        "symfony/http-foundation": "^6.0 || ^7.0",
-        "symfony/http-kernel": "^6.0 || ^7.0",
-        "symfony/options-resolver": "^6.0 || ^7.0"
+        "symfony/lock": "^6.0 || ^7.0 || ^8.0",
+        "symfony/cache": "^6.0 || ^7.0 || ^8.0",
+        "symfony/http-foundation": "^6.0 || ^7.0 || ^8.0",
+        "symfony/http-kernel": "^6.0 || ^7.0 || ^8.0",
+        "symfony/options-resolver": "^6.0 || ^7.0 || ^8.0"
     },
     "require-dev": {
-        "symfony/phpunit-bridge": "^6.0 || ^7.0"
+        "symfony/phpunit-bridge": "^6.0 || ^7.0 || ^8.0"
```

i got it to work locally with this:

```suggestion
        "symfony/phpunit-bridge": "^7.4 || ^8.0"
```

older phpunit-bridge always led to some class loader error with `Trait "Symfony\Bridge\PhpUnit\Legacy\PolyfillAssertTrait" not found` (the trait was present). imho this is good enough as its only a dev dependency and won't affect users. i verified that --prefer-lowest still installs all version 6 dependencies, so phpunit-bridge 7 does not force any other dependencies to be higher than otherwise.